internal/installation/paths_windows.go

```diff
@@ -13,3 +13,7 @@ func defaultSystemInstallPath() (string, error) {
 	// There is no system install path for Windows
 	return "", nil
 }
+
+func InstallRegistryKeyPath() string {
+	return `SOFTWARE\ActiveState\install`
+}
```

Comment:
Can we make this a constant? Note it's only used on Windows so doesn't need to exist on Linux/Mac.